src/sentry/sentry_metrics/kafka_metrics_interface.py

```diff
@@ -0,0 +1,42 @@
+from typing import Mapping
+
+from sentry.sentry_metrics.metrics_interface import GenericMetricsBackend
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class KafkaGenericMetricsBackend(GenericMetricsBackend):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def counter(
```

Comment:
Are there use cases where we want the count, set and distribution for a metric? Or does user have to always pick one of the 3

Reply:
Sure, there could be scenarios in which we would want the count, set, and distribution for a metric, but doesn't this support that? A user could instantiate the implementation class (e.g. Kafka) and then send each of the metric types.